src/mcs/lhs.jl

```diff
@@ -169,7 +169,7 @@ function lhs!(sim::LatinHypercubeSimulation; corrmatrix::Union{Matrix{Float64},N
     trials = sim.trials
     rvdict = sim.rvdict
     num_rvs = length(rvdict)
-    rvlist = sim.dist_rvs
+    rvlist = collect(values(sim.rvdict))
```

Comment:
I just add this extra `collect` because the `lhs` function that gets called below iexpects a vector of RVs, not an iterator. All of the other places where you replace `sim.dist_rvs` with `values(sim.rvdict)` work though because they only get used in for loops